src/commands/createNewProject/ProjectCreateStep/PysteinProjectCreateStep.ts

```diff
@@ -19,6 +20,16 @@ import { showMarkdownPreviewFile } from '../../../utils/textUtils';
 
 const gettingStartedFileName = 'getting_started.md';
 
+async function fileExists(uri: vscode.Uri): Promise<boolean> {
```

Comment:
You could use [this](https://github.com/microsoft/vscode-azuretools/blob/main/utils/src/utils/AzExtFsExtra.ts#L17-L21) to save some lines, though I will go back and add a `fileExists` utility function later.

Reply:
You would also not have to wrap the file paths with `Uri.file()` as well.

Reply:
Doesn't `stat()` throw if the file *doesn't* exist?